src/Mutant/Exception/ParserException.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Mutant\Exception;
+
+/**
+ * @internal
+ */
+final class ParserException extends \Exception
+{
+    public static function fromInvalidFile(\SplFileInfo $file, \Throwable $original): self
```

Comment:
subjective but I find importing symbols instead of leading `\` more readable

Reply:
That is what we have been doing in other classes as well.